osbs/build/build_request.py

```diff
@@ -519,8 +504,8 @@ def adjust_for_registry_api_versions(self):
         """
         versions = self.spec.registry_api_versions.value
 
-        if 'v2' not in versions:
-            raise OsbsValidationException('v1-only docker registry API is not supported')
+        if 'v1' in versions:
+            raise OsbsValidationException('v1 docker registry API is not supported')
```

Comment:
This is no longer being covered by the test suite

Reply:
Thanks, fixed.